app/frontend/components/EnrollmentIndex.jsx

```diff
@@ -37,83 +34,89 @@ function EnrollmentIndex(props) {
   };
 
   function DisplayDeleteSuccess(confirmation) {
-    if(confirmation == 'true') {
+    if (confirmation === 'true') {
```

Comment:
Good catch with the loose equalities I'd set! We do indeed need the strict ones!